app/Http/Backend/Requests/CreateProductsRequest.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace App\Http\Backend\Requests;
+
+use App\Http\Backend\Requests\Request;
+use Route;
+
+class CreateProductsRequest extends Request
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        return [
+            'name'=>'required|unique:products,name',
+            'quantity'=>'required',
+            'price'=>'required',
+            'image'=>'required|image|max:2048|',
+        ];
+    }
+
+    /**
+     * Message validate
+     *
+     * @return array
+     */
+    public function messages()
+    {
+        return [
```

Comment:
You use `English` but @at-tienhx1 use `Vietnamese`. So I will ask @at-tienhx1 to change to use `English`. 
And `English` is default language of Laravel so you don't need to define validation message again